CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.0.0]
+### Uncategorized
```

Comment:
We'll need to update the changelog, this should be failing the linter (CI hasn't run here because the create-release-PR job is broken in that it won't run checks until after at least one commit is made).

We'll want to ensure that we highlight that this is a breaking change, and explain in detail why and what has changed.